frontend/src/features/builder/EditorContext.tsx

```diff
@@ -70,7 +71,19 @@ export const useProvideEditor = (): EditorContextProps => {
     }
   }, [value])
 
+  // https://github.com/udecode/plate/issues/1349
+  const resetEditor = useCallback(async () => {
```

Comment:
This function hard wipes the editor state by editor id in the plate store, but somehow it STILL displays a stale state after a template is updated (the API call returns the updated state).